app/models/game.rb

```diff
@@ -66,6 +66,10 @@ def current_turn
     end
   end
 
+  def player_color(player)
```

Comment:
def player_color
    return true if white_turn?
    return false if black_turn?
end

Try using the above